src/zarr/chunk_key_encodings.py

```diff
@@ -38,7 +38,13 @@ def from_dict(cls, data: dict[str, JSON] | ChunkKeyEncoding) -> ChunkKeyEncoding
         if isinstance(data, ChunkKeyEncoding):
             return data
 
-        name_parsed, configuration_parsed = parse_named_configuration(data)
+        # configuration is optional for chunk key encodings
+        name_parsed, configuration_parsed = parse_named_configuration(
+            data, require_configuration=False
+        )
+        # normalize missing configuration to the default "/" separator.
```

Comment:
The default for `default` is `/`. For `v2` it is `.`

Reply:
good catch, I fixed this in [404b60a](https://github.com/zarr-developers/zarr-python/pull/2058/commits/404b60aa65b3c6a07634557d5b8ddce926c2a285)